esm/streams/application-logs.js

```diff
@@ -88,7 +88,7 @@ export class ApplicationLogStream extends CleverCloudSse {
   /**
    * shortcut for .on('APPLICATION_LOG', (event) => ...)
    * @param {Function} fn which handle logs
-   * @returns {any}
+   * @returns {ApplicationLogStream}
```

Comment:
nitpick: you can do `@returns {this}` instead.

fix: can you move that into a separate commit?